README.md

```diff
@@ -15,7 +15,7 @@ Poise is an opinionated Discord bot framework with a few distinctive features:
 # How to use
 
 Most information is in the [API documentation](https://docs.rs/poise/). Also take a
-look at the [examples](examples), especially [`feature_showcase`](https://github.com/ev3nvy/poise/tree/current/examples/feature_showcase), to learn what poise can do.
```

Comment:
Wow this is wild, I have no idea how this link ended up in here xD It's the fork from a minor contributor. Thanks for noticing